docs/examples/models_default_factory.py

```diff
@@ -1,9 +1,12 @@
-from pydantic import BaseModel, Field
+from datetime import datetime
 from uuid import UUID, uuid4
+from pydantic import BaseModel, Field
 
 class Model(BaseModel):
     uid: UUID = Field(default_factory=uuid4)
+    updated: datetime = Field(default_factory=datetime.utcnow)
 
 m1 = Model()
 m2 = Model()
 assert m1.uid != m2.uid
+assert m1.updated != m2.updated
```

Comment:
better to print here than do an assert, could you change the case above too?

Reply:
Sure thing.